crates/transpiler/src/passes/constrained_reschedule.rs

```diff
@@ -0,0 +1,304 @@
+// This code is part of Qiskit.
+//
+// (C) Copyright IBM 2025.
+//
+// This code is licensed under the Apache License, Version 2.0. You may
+// obtain a copy of this license in the LICENSE.txt file in the root directory
+// of this source tree or at http://www.apache.org/licenses/LICENSE-2.0.
+//
+// Any modifications or derivative works of this code must retain this
+// copyright notice, and modified files need to carry a notice indicating
+// that they have been altered from the originals.
+
+use crate::target::Target;
+use crate::TranspilerError;
+use ::hashbrown::HashSet;
```

Comment:
A minor style thing you don't need the leading `::` here.
```suggestion
use hashbrown::HashSet;
```